File: auth/src/main/java/com/diagorn/sparkathon/auth/dto/auth/LoginRequest.java
package com.diagorn.sparkathon.auth.dto.auth;

import lombok.Data;
import lombok.ToString;

/**
 * Login request
 *
 * @author diagorn
 */
@Data
@ToString
public class LoginRequest {
    /**
     * Username
     */
    private String login;
    /**
     * User password
     */
    private String password;
}

File: auth/src/main/java/com/diagorn/sparkathon/auth/dto/auth/LoginRequest.kt
package com.diagorn.sparkathon.auth.dto.auth

import io.swagger.v3.oas.annotations.media.Schema
import io.swagger.v3.oas.annotations.media.Schema.RequiredMode

/**
 * Login request
 *
 * @author diagorn
 */
class LoginRequest(
    /**
     * Username
     */
    @Schema(description = "Username", example = "diagorn", requiredMode = RequiredMode.REQUIRED)
    val login: String,

    /**
     * User password
     */
    @Schema(description = "password", example = "qwe123", requiredMode = RequiredMode.REQUIRED)
    val password: String
)